src/main/java/com/excella/foodinc/data/FoodData.java

```diff
@@ -0,0 +1,23 @@
+package com.excella.foodinc.data;
+
+import java.util.HashMap;
+import com.excella.foodinc.models.Food;
+
+public class FoodData {
+

```

Comment:
You might not need this class, you could put all this in a constants file somewhere and then just reference that constant in your service.